_data/chains/eip155-690.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "Redstone",
+  "chain": "ETH",
+  "rpc": ["https://rpc.redstonechain.com", "wss://rpc.redstonechain.com"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "infoURL": "https://redstone.xyz",
+  "shortName": "redstone",
+  "chainId": 690,
+  "networkId": 690,
+  "icon": "redstone",
+  "explorers": [
+    {
+      "name": "blockscout",
+      "url": "https://explorer.redstone.xyz",
+      "icon": "blockscout",
+      "standard": "EIP3091"
+    }
+  ],
+  "parent": {
+    "type": "L2",
+    "chain": "eip155-1",
+    "bridges": [
+      {
+        "url": "https://redstone.xyz/deposit"
```

Comment:
can we add the same for garnet?

Reply:
done, added https://garnetchain.com/deposit